packages/react-charts/src/components/ChartBar/ChartBar.tsx

```diff
@@ -422,12 +422,20 @@ export interface ChartBarProps extends VictoryBarProps {
    * @example 'last_quarter_profit', () => 10, 1, 'employees.salary', ["employees", "salary"]
    */
   y0?: DataGetterPropType;
+  /**
+   * Use ariaLabel prop to determine how screen reader will read the contents in the chart.
+   * If you don't define this prop, the content in the chart will not be accessible.
+   *
+   * @example ['There are 7 cats in 2015', 'There are 5 cats in 2016', 'There are 8 cats in 2017', 'There are 2 cats in 2018']
+   */
+  ariaLabel?: string[];
```

Comment:
I don't believe we want to pass in an array of strings here. In Victory's Bar example, they provide the datum prop as shown below:

ariaLabel={({ datum }) => x: ${datum.x}}

We follow this same pattern for our tooltips, for example.

The `datum` property can contain a lot of information. Besides just the `x` prop shown above, devs can add their own custom properties, which is very useful for creating custom content.

Reply:
This should also be in alphabetical order.

Reply:
Fixed, thanks!